db/dataset.py

```diff
@@ -315,3 +315,54 @@ def _delete_snapshots_for_dataset(connection, dataset_id):
         DELETE FROM dataset_snapshot
               WHERE dataset_id = :dataset_id""")
     connection.execute(query, {"dataset_id": dataset_id})
+
+
+def add_recordings(dict, dataset_id):
+    """Adds new recordings to a class in a dataset"""
+
+    with db.engine.begin() as connection:
+        result = connection.execute(sqlalchemy.text("""SELECT id FROM dataset_class WHERE name = :name and dataset = :dataset_id"""),
+                                    {"name": dict["class_name"], "dataset_id": dataset_id})
+        if result.rowcount < 1:
+            raise exceptions.NoDataFoundException("No such class exists.")
+        clsid = result.fetchone()
+        for mbid in dict["recordings"]:
+            connection.execute(sqlalchemy.text("""INSERT INTO dataset_class_member (class, mbid) VALUES (:clsid, :mbid)"""), {"clsid": clsid[0], "mbid": mbid})
+
+
+def delete_recordings(dict, dataset_id):
+    """Delete all recordings from the list of IDs"""
+
+    with db.engine.begin() as connection:
+        result = connection.execute(sqlalchemy.text("""SELECT id FROM dataset_class WHERE name = :name and dataset = :dataset_id"""),
```

Comment:
You use this query in all of your methods, to get the class. You can move this into a method
```python
def _get_class_in_dataset(connection, dataset_id, class_name)
```
so that you don't duplicate code.  You can then test this method separately.
The test should check four things:
 - if the dataset doesn't exist, raises an error
 - if the dataset exists but the class doesn't, raise an error
 - if the dataset and class exist, return it
 - if there are two datasets and they both have a class called "Class #1", calling this method will get the class from the correct dataset